### PR TITLE
Deprecation of WebSocket endpoints

### LINKED ISSUE #1622: Deprecation of WebSocket endpoints
### URL, if applicable

https://docs.quantum.ibm.com/start/latest-updates

### Describe the new content you are requesting.

Due to low usage, we will be removing the WebSocket endpoints:
- `GET /stream/jobs`
- `GET /stream/jobs/{id}`

The `job.stream_results()` and `job.interim_results()` methods in `qiskit-ibm-runtime` use these endpoints and therefore will no longer work once the endpoints are removed, regardless of the `qiskit-ibm-runtime` version. `job.result()` also uses these endpoints, and users will need to upgrade to `qiskit-ibm-runtime` 0.25 or later before the endpoints are removed.

These features will be removed no sooner than 3 months from now, after which point users will no longer be able to stream intermediate results and job status.

The deprecation period will start when this announcement is live, ideally the week of July 1.

### If this new content request is accepted, do you want to write the content?

I can help the team by providing enough information to write the material

### PR DESCRIPTION
**Pull request by beckykd**
closes #1622

### CODE REVIEW

**Comment by beckykd** [comment]
FYI @drew-distefano - I can't add you as a reviewer, for some reason.

File: docs/start/latest-updates.mdx [hunk 1]
```diff
@@ -76,7 +76,14 @@ On the IBM Quantum blog, we take a deep dive into the new beta release with a sp
 
 ## Qiskit IBM Runtime service updates
 
-This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use. 
+This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use.
+
+### 1 July 2024
+
+The following endpoints are deprecated and will be removed on or after 1 October, 2024: `GET /stream/jobs` and `GET /stream/jobs/{id}` This removal has the following impacts:
```
Comment (abbycross) [inline docs/start/latest-updates.mdx:83]:
```suggestion
The following endpoints are deprecated and will be removed on or after 1 October 2024: `GET /stream/jobs` and `GET /stream/jobs/{id}` This removal has the following impacts:
```

File: docs/start/latest-updates.mdx [hunk 2]
```diff
@@ -76,7 +76,14 @@ On the IBM Quantum blog, we take a deep dive into the new beta release with a sp
...
 ## Qiskit IBM Runtime service updates
 
-This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use. 
+This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use.
+
+### 1 July 2024
+
+The following endpoints are deprecated and will be removed on or after 1 October 2024: `GET /stream/jobs` and `GET /stream/jobs/{id}` This removal has the following impacts:
+
+- After the endpoints are removed, `job.stream_results()` and `job.interim_results()` will be removed.
```
Comment (Eric-Arellano) [inline docs/start/latest-updates.mdx:85]:
```suggestion
- After the endpoints are removed, `job.stream_results()` and `job.interim_results()` will be removed from the [qiskit-ibm-runtime](/api/qiskit-ibm-runtime) client.
```

File: docs/start/latest-updates.mdx [hunk 3]
```diff
@@ -76,7 +76,14 @@ On the IBM Quantum blog, we take a deep dive into the new beta release with a sp
...
 ## Qiskit IBM Runtime service updates
 
-This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use. 
+This summary of the latest changes to the Qiskit Runtime service applies to anyone using Qiskit Runtime, regardless of how you communicate with the service (using qiskit-ibm-runtime or otherwise), or which version of the client SDK you use.
+
+### 1 July 2024
+
+The following endpoints are deprecated and will be removed on or after 1 October 2024: `GET /stream/jobs` and `GET /stream/jobs/{id}`. This removal has the following impacts:
+
+- After the endpoints are removed, `job.stream_results()` and `job.interim_results()` will be removed from the [qiskit-ibm-runtime](/api/qiskit-ibm-runtime) client.
```
Comment (jyu00) [inline docs/start/latest-updates.mdx:85]: Do we want to link to the API docs for these methods?

File: docs/start/latest-updates.mdx [hunk 4]
```diff
@@ -76,7 +76,14 @@ On the IBM Quantum blog, we take a deep dive into the new beta release with a sp
...
+- Additional methods, such as `job.result()` currently use the deprecated endpoints.  Upgrade to `qiskit_ibm_runtime` 0.25 or later before the endpoints are removed to avoid disruption.
```
Comment (jyu00) [inline docs/start/latest-updates.mdx:86]: So we don't have qiskit-ibm-runtime 0.25 yet 😅. The 2 PRs needed for this were merged, so @kt474 can presumably make a release tomorrow. Do we want to wait for that?

Comment (kt474) [reply docs/start/latest-updates.mdx:86]: Sure, I can do a release tmrw

Comment (beckykd) [reply docs/start/latest-updates.mdx:86]: That seems sensible

File: docs/guides/latest-updates.mdx [hunk 5]
```diff
@@ -5,7 +5,7 @@ description: The latest updates from Qiskit and IBM Quantum, including the lates
 
 # Latest updates
 
-*Last updated 27 June 2024*
+*Last updated 1 July 2024*
```
Comment (abbycross) [inline docs/guides/latest-updates.mdx:8]:
```suggestion
*Last updated 2 July 2024*
```

**Review by abbycross — APPROVED** [review]
👍

File: docs/guides/latest-updates.mdx [hunk 6]
```diff
@@ -119,10 +119,10 @@ IBM Quantum documentation recently added a number of user-facing improvements, i
...
 **New pages**
 
-- [Error mitigation techniques](./error-mitigation-and-suppression-techniques)
-- [Error mitigation and suppression techniques](./error-mitigation-and-suppression-techniques)
-- [The Operator class](./operator-class)
-- [Transpile against custom backends](./custom-backend)
+- [Error mitigation techniques](/run/error-mitigation-explanation)
+- [Error mitigation and suppression techniques](/run/error-mitigation-explanation)
+- [The Operator class](/build/operator-class)
+- [Transpile against custom backends](/transpile/custom-backend)
```
Comment (Eric-Arellano) [inline docs/guides/latest-updates.mdx:125]: Why are the first two files the same link?

```suggestion
- [Error mitigation techniques](./error-mitigation-and-suppression-techniques)
- [Error mitigation and suppression techniques](./error-mitigation-and-suppression-techniques)
- [The Operator class](./operator-class)
- [Transpile against custom backends](./custom-backend)
```

**Review by Eric-Arellano — APPROVED** [review]

**Pull request merged**